### PR TITLE
Select country on registration

### PR DESCRIPTION
**Pull request by squirrelo**
This makes the country list a selection instead of free text for registering a kit. This way at least we have controlled vocab for country of registration for later use in survey pulldown inferences.

~~Requires #462 first to avoid merge conflicts~~

### CODE REVIEW

**Comment by squirrelo** [comment]
Ready for review

File: amgut/templates/register_user.html [hunk 1]
```diff
@@ -83,7 +83,14 @@
             <tr><td>{% raw tl['CITY'] %}</td><td><input type="text" name="city" id="city"></td></tr>
             <tr><td>{% raw tl['STATE'] %}</td><td><input type="text" name="state" id="state"></td></tr>
             <tr><td>{% raw tl['ZIP'] %}</td><td><input type="text" name="zip" id="zip"></td></tr>
-            <tr><td>{% raw tl['COUNTRY'] %}</td><td><input type="text" name="country" id="country"></td></tr>
+            <tr><td>{% raw tl['COUNTRY'] %}</td><td>
+              <select name="country" id="country">
+              <option value=''></option>
+              {% for country in countries %}
+                <option value='{{country}}'>{{country}}</option>
```
Comment (wasade) [inline amgut/templates/register_user.html]: possible to put US and UK up at the top and/or make this searchable?

Comment (squirrelo) [inline amgut/templates/register_user.html]: I can make it a chosen dropdown

**Comment by squirrelo** [comment]
Done, ready for another review.

**Comment by adamrp** [comment]
:+1:

**Comment by squirrelo** [comment]
Added a patch, so needs another review.

**Comment by wasade** [comment]
@squirrelo, conflicts

**Comment by wasade** [comment]
:+1: otherwise

**Comment by squirrelo** [comment]
conflict resolved, if still :+1: then merge ready.

**Pull request merged**